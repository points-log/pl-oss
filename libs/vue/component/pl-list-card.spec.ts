import { shallowMount } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';
import PLListCard from './pl-list-card.vue';

Vue.use(Vuetify);

describe('PLListCard', () => {
  it('should match the snapshot', () => {
    const propsData = { items: [], loading: false, routeName: 'routeName' };
    const wrapper = shallowMount(PLListCard, { propsData });
    expect(wrapper.element).toMatchSnapshot();
  });
});
