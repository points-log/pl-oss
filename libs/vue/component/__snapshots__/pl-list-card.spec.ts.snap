// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLListCard should match the snapshot 1`] = `
<v-card-stub
  loaderheight="4"
  tag="div"
>
  <v-card-text-stub
    class="text-center"
  >
    No data available
  </v-card-text-stub>
</v-card-stub>
`;
